### DISCUSSION TITLE
Feature request:  Prometheus - Disable exemplars by default

### DISCUSSION THREAD

**Discussion by dbluxo** [grafana/grafana#44264] [Data Sources]
As came up from [this PR](https://github.com/grafana/grafana/pull/42749#issuecomment-1016374263) I would prefer to set exemplars to `false` by default and only toggle it for specific queries where I need it to. Otherwise, queries which do not return exemplars will cause error messages in Grafana `msg="Exemplar query failed"` by default.

**Comment by ivanahuckova** [comment]
Fixed by https://github.com/grafana/grafana/pull/45260 that is part of 8.5.0-beta1 release. Exemplars now default to false.
